### ISSUE TITLE
Vehicle Position Data Not Updating in Home Assistant Integration

### ISSUE THREAD

**Issue by jmttramos** [flobz/psa_car_controller#956]
**Describe the bug**
The integration with Home Assistant stopped updating the vehicle's position. Other features (e.g., preconditioning) are still working correctly. Accessing the endpoint `http://127.0.0.1/get_vehicleinfo/[VIN Number]` shows the last position date as 31/08/2024, even though the preconditioning date updates correctly to the current date.

**To Reproduce**
1. Set up the integration with Home Assistant.
2. Check the vehicle position data.
3. Observe that the position's `updated_at` date does not reflect the current date.

**Expected behavior**
The vehicle's position should update with the current date and time.

**Vehicle Info**
- Endpoint: `http://127.0.0.1/get_vehicleinfo/[VIN Number]`
- Last Position Data: `"updated_at": "2024-08-31 20:03:38+00:00"`
- Preconditioning Data: `"updated_at": "2024-09-14 10:26:39+00:00"`

**Additional context**
- Home Assistant version: 2024.09.1
- PSA Car Controller version: v3.5.3

**Comment by R-dehaan** [comment]
Did you use eevee in the past to get it working?

**Comment by jmttramos** [comment]
Yes, I did use Eevee in the past, and it worked very well. However, since they switched to a subscription model, it hasn’t worked as smoothly as before. Now, I only receive updates in my Home Assistant map intermittently.

**Comment by R-dehaan** [comment]
Yes so that confirms my suspicion that after a while stellantis stops sending detailed data via the API.

Don't know why 3rd party applications enable to stellantis to sent these responses back. @flobz is there any change we can have similar responses back, without paying for e.g Eevee?

**Comment by jmttramos** [comment]
It’s likely they had to cover costs for accessing vehicle position data from Stellantis, which could have contributed to their shift to a subscription model. This aligns with their announcement, where they cite increased licensing fees and a growing user base as factors behind the change. Their full statement is [here](https://intercom.help/eevee-mobility/en/articles/9659645-introducing-our-pro-subscription-for-the-eevee-driver-app). 
Just a thought!
